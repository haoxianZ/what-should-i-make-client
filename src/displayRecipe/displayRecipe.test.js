import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import DisplayRecipe from './displayRecipe'

describe(`Display searched result component`, () => {

  it('renders search recipe results', () => {
    const wrapper = shallow(<DisplayRecipe />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
