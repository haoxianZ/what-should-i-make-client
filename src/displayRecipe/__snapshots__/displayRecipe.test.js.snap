// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display searched result component renders search recipe results 1`] = `
<div
  className="recipes"
>
  <form
    onSubmit={[Function]}
  >
    <label
      htmlFor="filter"
    >
      Choose a filter:
    </label>
    <select
      id="filter"
      name="filter"
    >
      <option
        value=""
      >
        None
      </option>
      <option
        value="alcohol-free"
      >
        Alcohol free
      </option>
      <option
        value="peanut-free"
      >
        Peanut free 
      </option>
      <option
        value="sugar-conscious"
      >
        Sugar conscious 
      </option>
      <option
        value="tree-nut-free"
      >
        Tree nut free 
      </option>
      <option
        value="vegan"
      >
        Vegan
      </option>
      <option
        value="vegetarian"
      >
        Vegetarian
      </option>
    </select>
    <button
      className="submitBtn"
      type="submit"
    >
      Search
    </button>
  </form>
  <LoadMore
    click={0}
  />
  <button
    onClick={[Function]}
  >
    Load More Results
  </button>
  <button
    id="myBtn"
    onClick={[Function]}
    title="Go to top"
  >
    Top
  </button>
</div>
`;
